README.md

```diff
@@ -144,6 +144,8 @@ The cmdlet does not fully support the Inquire action for debug messages. Cmdlet
 ### Unreleased
 
 * Templates folder removed. Use the test templates in the [Tests.Template folder in the DSCResources repository](https://github.com/PowerShell/DscResources/tree/master/Tests.Template) instead.
```

Comment:
Need to add a description of the resources and its properties, see line 121 for xNetworkTeam
and an example line 260 after your changes starts the examples.